Sources/Jetworking/Client/Client.swift

```diff
@@ -375,27 +385,36 @@ public final class Client {
         }
     }
 
+    private func handleSuccessVoid<ResponseType>(currentURLResponse: HTTPURLResponse, endpoint: Endpoint<ResponseType>?, data: Data?, completion: @escaping RequestCompletion<ResponseType>) {
+        if ResponseType.self is Void.Type {
```

Comment:
Can we refactor it to a guard?

Reply:
Yes we can. Though this method is now moved to a dedicated file